README.md

```diff
@@ -1,2 +1,42 @@
-# group-Jessica
-Household Manager Web Application using C#
+# Household Pro
+
+## Description
+Household Manager Web Application using C# and ASP.NET Core 6 via MVC created with the intent for 5 entry-level programmers to practice building an application from scratch utilizing Agile development and version-control via branching in GitHub.
+
+Household Pro allows individuals that live in a multi-person unit to collect, organize, and assign all chores and assignments involved in keeping up with home organization and cleanliness efficiently and faily.
```

Comment:
fairly (correct misspelling of faily at the end of the sentence)

Reply:
Change made.